modules/bootstrap-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+// * networking/installing-albo-sts-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="nw-bootstra-albo-on-sts-cluster_{context}"]
+= Bootstrapping AWS Load Balancer Operator on Secure Token Service cluster
+
+.Prerequisites
+
+* You must extract and prepare the `coctl` binary.
+
+.Procedure
+
+The AWS Load Balancer Operator relies on `CredentialsRequest` to bootstrap the operator. The Cloud Credential Operator does not provision the secrets automatically in the STS cluster. You must set the credentials secrets manually by using the `ccoctl` binary. The AWS Load Balancer Operator waits until the required secrets are created and available.
```

Comment:
We don't need this note anymore as it's on the top now.

Reply:
yes, forgot to push that change. acked